### ISSUE TITLE
Interactive velocity analysis code (iva.sh and iva.scr) not working

### ISSUE THREAD

**Issue by tgautam03** [JohnWStockwellJr/SeisUnix#52]
I am trying to do Interactive velocity analysis. I've iva.sh and iva.scr. When I am trying to open interactive dialogue window using iva.scr, it's not doing anything and giving output like 
`(base) atom@Blade:~/work/mdg/su$ ./iva.scr &`
`[1] 16516`
but when I am running iva.sh directly it's giving errors like 
![Screenshot from 2019-12-05 16-47-51](https://user-images.githubusercontent.com/30533136/70230809-2529c000-177f-11ea-89ec-d16a3a6ccf39.png)

Code files are as follows : 
[iva_codes.zip](https://github.com/JohnWStockwellJr/SeisUnix/files/3926516/iva_codes.zip)

**Comment by JohnWStockwellJr** [comment]
I am unfamiliar with the iva scripts. I assume you got those from Forel’s book.
Try using Velan.radon from the Course demo directory.

John Stockwell
Visit: https://wiki.Seismic-Unix.org
The new home of Seismic Un*x

**Comment by tgautam03** [comment]
Yes, I am working through that book and trying to make a workflow for velocity analysis. 
I can't find the course demo directory anywhere that you just mentioned. 
Can you also suggest some place where I can post my doubts like this for future?

**Comment by JohnWStockwellJr** [comment]
If you send me your direct email address I will send you Dropbox links for my course materials.

John Stockwell
Visit: https://wiki.Seismic-Unix.org
The new home of Seismic Un*x

**Comment by JohnWStockwellJr** [comment]
$CWPROOT/src/demos/Course

John Stockwell
Visit: https://wiki.Seismic-Unix.org
The new home of Seismic Un*x

**Comment by tgautam03** [comment]
It will be great you can share your course material. My email is : tgautam@es.iitr.ac.in
Thanks

**Comment by Nightwolf86** [comment]
Hello Sir

I am also using the iva.sh script for velocity analysis. My script is working but I have a problem with the semblance plot. The semblance plot's time axis goes down to 0.6s. On the other hand,  the time of the other two plots, which are constant velocity stacks and CMP gather, go down 3s. The semblance plot's time must be 3s like the others. How can I solve this problem? 

Can I also get your course materials?
My mail -> eapatay@mtu.edu

![velocity](https://user-images.githubusercontent.com/57058564/73432393-a7160f80-4310-11ea-959b-7e84593d6303.jpg)

Thanks
Best regards

**Comment by sanmurilo** [comment]
I'm working on the iva script, I managed to remove the syntax errors, but there is still an error in the sunmo function. If you have any questions, I'm contributing to the https://github.com/Dirack/proSU repository. My email: murilovj@gmail.com.
![CMP_1150](https://user-images.githubusercontent.com/23323271/78203931-a283e980-746e-11ea-896b-202fa0aa06a1.png)
![CMP_After_NMO_1150](https://user-images.githubusercontent.com/23323271/78203932-a3b51680-746e-11ea-8644-9e43a2ddd571.png)
![Semblance_1150](https://user-images.githubusercontent.com/23323271/78203933-a44dad00-746e-11ea-80bf-e690341e6d68.png)
![Screenshot from 2020-04-01 23-02-29](https://user-images.githubusercontent.com/23323271/78203851-70728780-746e-11ea-9f28-999c796f2179.png)
![Screenshot from 2020-04-01 23-03-36](https://user-images.githubusercontent.com/23323271/78203857-76686880-746e-11ea-88c5-ab6b97f176d2.png)
![Screenshot from 2020-04-01 23-05-27](https://user-images.githubusercontent.com/23323271/78203873-7cf6e000-746e-11ea-926f-cdf6c2cac689.png)
[iva.zip](https://github.com/JohnWStockwellJr/SeisUnix/files/4418544/iva.zip)
